[2022-03-18 04:46:24,357] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: pbp_bucket_ingestion_dag.get_pbp_data scheduled__2014-11-20T04:00:00+00:00 [queued]>
[2022-03-18 04:46:24,424] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: pbp_bucket_ingestion_dag.get_pbp_data scheduled__2014-11-20T04:00:00+00:00 [queued]>
[2022-03-18 04:46:24,426] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 04:46:24,428] {taskinstance.py:1239} INFO - Starting attempt 2 of 2
[2022-03-18 04:46:24,429] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 04:46:24,478] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_pbp_data> on 2014-11-20 04:00:00+00:00
[2022-03-18 04:46:24,501] {standard_task_runner.py:52} INFO - Started process 560 to run task
[2022-03-18 04:46:24,510] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'pbp_bucket_ingestion_dag', 'get_pbp_data', 'scheduled__2014-11-20T04:00:00+00:00', '--job-id', '6283', '--raw', '--subdir', 'DAGS_FOLDER/pbp_ingestion_dag.py', '--cfg-path', '/tmp/tmpa10tsyp2', '--error-file', '/tmp/tmp_qfd190_']
[2022-03-18 04:46:24,513] {standard_task_runner.py:77} INFO - Job 6283: Subtask get_pbp_data
[2022-03-18 04:46:24,730] {logging_mixin.py:109} INFO - Running <TaskInstance: pbp_bucket_ingestion_dag.get_pbp_data scheduled__2014-11-20T04:00:00+00:00 [running]> on host a2ba0547c319
[2022-03-18 04:46:24,875] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=pbp_bucket_ingestion_dag
AIRFLOW_CTX_TASK_ID=get_pbp_data
AIRFLOW_CTX_EXECUTION_DATE=2014-11-20T04:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2014-11-20T04:00:00+00:00
[2022-03-18 04:46:25,104] {python.py:175} INFO - Done. Returned value was: None
[2022-03-18 04:46:25,181] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=pbp_bucket_ingestion_dag, task_id=get_pbp_data, execution_date=20141120T040000, start_date=20220318T044624, end_date=20220318T044625
[2022-03-18 04:46:25,262] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-18 04:46:25,396] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
